pkg/tfvars/azure/azure.go

```diff
@@ -2,7 +2,6 @@ package azure
 
 import (
 	"encoding/json"
-
```

Comment:
keep the line here, the go imports are best organised
```
# stdlib packages

# external packages

# internal installer repo packages
```

Reply:
Fixed.